src/Serilog/Formatting/Display/MessageTemplateTextFormatter.cs

```diff
@@ -91,6 +91,15 @@ public void Format(LogEvent logEvent, TextWriter output)
 
                     token.Render(overridden, output, _formatProvider);
                 }
+                else if (pt.IsLevelProperty)
```

Comment:
Instead of doing this here, we can just return a `LogEventLevelValue` from https://github.com/serilog/serilog/blob/dev/src/Serilog/Formatting/Display/OutputProperties.cs#L68